src/stock-physical-inventory-draft/physical-inventory-draft.html

```diff
@@ -0,0 +1,147 @@
+<h2>
+    {{'stockPhysicalInventoryDraft.title' | message: {'facilityCode': vm.facility.code, 'facilityName': vm.facility.name, 'program': vm.program.name} }}
+</h2>
+
+<!-- ANGOLASUP-806: Implement adding default reason in physical inventory-->
+<div class="toolbar" ng-style="{'width': vm.getToolbarWidth()}">
+    <div  class="pull-right" ng-style="{'display': 'flex', 'align-items': 'center', 'gap': '0.5em', 'height': '1em'}">
+        <label for="useDefaultReason">
+            <input id="useDefaultReason"
+                    type="checkbox"
+                    name="useDefaultReason"
+                    ng-true-value="true"
+                    ng-false-value="false"
+                    ng-model="vm.useDefaultReason"
+                    ng-change="vm.updateUseDefaultReason()"/>
+
+            {{'stockPhysicalInventoryDraft.useDefaultReason' | message}}
```

Comment:
This could be optimized a little bit. Difference isn't big but it's worth sticking to the best practices.

It's a static text that needs to be translated which won't change when we are on this view. We can make Angular evaluate this expression only once instead of checking it on every digest cycle. To achieve that you can add `::` to the expression so it's evaluated only once

`{{:: 'stockPhysicalInventoryDraft.useDefaultReason' | message}}`

https://docs.angularjs.org/guide/expression#one-time-binding

Reply:
I updated the code